src/main/java/dk/kb/util/YAMLUtils.java

```diff
@@ -0,0 +1,61 @@
+package dk.kb.util;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.Properties;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class YAMLUtils {
+    
+    /**
+     * Converts the YAML object into a properties object
+     *
+     * @param collection the yaml
+     * @return the yaml as properties
+     */
+    public static Properties toProperties(YAML collection) {
+        Map<String, Object> propertiesMap = toFlatList(collection, null)
+                                                    .collect(Collectors.toMap(
+                                                            tuple2 -> tuple2._1,
+                                                            tuple2 -> tuple2._2
+                                                    ));
+        
+        Properties properties = new Properties();
+        
+        properties.putAll(propertiesMap);
+        return properties;
+    }
+    
+    protected static Stream<Tuple2<String, Object>> toFlatList(Map<String, Object> collection, String keyFix) {
```

Comment:
This is not tied specifically to YAML, so maybe it should be in another class? `CollectionUtils` or something like that?

Reply:
Well... Perhaps. I have a lot of utils I would like to add, but the kb-utils is strongly tied to YAML parsing and nothing else

Reply:
That is only because YAML was the first thing we needed that we did not have in the old `sb-util`.